src/lib/components/colors/Header.svelte

```diff
@@ -80,6 +82,12 @@
         break;
     }
   };
+  const switchColorsT = () => {
+    switchColors();
+  };
```

Comment:
Why do we need to wrap this?

Reply:
It wasn't needed (but now it is, to make it conditional)